I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 268, in script
    return
Exception: Could not find return label ('game/script.rpy', 1696054856, 8).

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 268, in script
    return
  File "C:\Users\DiAko\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1801, in execute
    next_node(renpy.game.context().lookup_return(pop=True))
  File "C:\Users\DiAko\Downloads\renpy-8.1.3-sdk\renpy\execution.py", line 741, in lookup_return
    raise Exception("Could not find return label {!r}.".format(self.return_stack[-1]))
Exception: Could not find return label ('game/script.rpy', 1696054856, 8).

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
What remains of Ken 1.0
Sun Oct  8 17:14:12 2023
